superduperdb/backends/base/artifacts.py

```diff
@@ -135,15 +136,21 @@ def delete_artifact(self, r: t.Dict):
         )
 
         for blob in blobs:
-            self._delete_bytes(blob.split(':')[-1])
+            try:
+                self._delete_bytes(blob.split(':')[-1])
+            except FileNotFoundError:
+                logging.warn(f'Blob {blob} not found in artifact store')
```

Comment:
It seems that the issue was caused by a non-robust replace operation that deleted non-existent IDs. This will be addressed in the next version.